### PR TITLE
route logs to logback

### PR DESCRIPTION
**Pull request by andrej-sajenko**
Hi,

i tried to route the log messages to logback as described
* https://logging.apache.org/log4j/2.x/maven-artifacts.html
* https://blog.espenberntsen.net/tag/logback/

can you take a short look at the georocket-server and check if that is what you expected?

Note: I removed the log level to check if there is any logger which i forgot

### CODE REVIEW

**Comment by michel-kraemer** [comment]
Yes, this is what I was looking for.

Just a question. When I start GeoRocket now (without your pull request) I get the following warning:

    log4j:WARN No appenders could be found for logger (io.netty.util.internal.logging.InternalLoggerFactory).
    log4j:WARN Please initialize the log4j system properly. 
    log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.

Will the PR fix this warning? If yes, perfect!

File: georocket-server/src/test/resources/logback-test.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (michel-kraemer) [inline georocket-server/src/test/resources/logback-test.xml]: What's the difference between this file and the logback.xml? Is this one used for unit tests?

Comment (andrej-sajenko) [inline georocket-server/src/test/resources/logback-test.xml]: I assume that grade behave like maven and use logback-test.xml only for unit tests. 

"If you are using Maven and if you place the logback-test.xml under the src/test/resources folder, Maven will ensure that it won't be included in the artifact produced. Thus, you can use a different configuration file, namely logback-test.xml during testing, and another file, namely, logback.xml, in production." 
Src: http://logback.qos.ch/manual/configuration.html

Comment (andrej-sajenko) [inline georocket-server/src/test/resources/logback-test.xml]: I try to fix the log configuration warnings before i mark the pull request as finished.

**Comment by andrej-sajenko** [comment]
I can not reproduce the log4j warnings (log4j:WARN).

**Comment by michel-kraemer** [comment]
Cool. Let me know when the PR is ready to be merged (when you've removed the [WIP] tag). Thanks!

**Comment by andrej-sajenko** [comment]
ready

**Comment by michel-kraemer** [comment]
Very good! Thanks!

**Pull request merged**